src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2243,10 +2243,6 @@ class PvtModelTableParametersDescription:
     number_of_phases: int = attr.ib(default=2, validator=instance_of(int))
     warn_when_outside: bool = attr.ib(default=True, validator=instance_of(bool))
 
-    table_type: Optional[str] = attr.ib(
```

Comment:
Removing this will cause existing alfacase files containing this key to be invalid.

Reply:
Yes, I have added this attribute before to identify the PH tables, but now it's no more required, after the refactoring.

Reply:
This was present only in dev versions, it should not be a problem